ix-dev/community/urbackup/templates/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.urbackup_container_name, "image") %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+{% do c1.add_caps(["CHOWN", "SETUID", "SETGID"]) %}
+{% do c1.healthcheck.set_test("http", {"port": values.network.web_port}) %}
+
+
+{% if values.urbackup.zfs_enabled %}
+  {% do c1.devices.add_device("/dev/zfs", "/dev/zfs") %}
+{% endif %}
```

Comment:
Hello,
With this here, this PR is not going to be merged.
if you think it is still of value to add it without this. Let me know.

Reply:
ill remove it. im currently running this on a non-truenas system with a zfs pool and this option disabled. I was just adapting what the author had in [his repo here](https://github.com/uroni/urbackup-server-docker)

Reply:
fixed